content/en/serverless/installation/go.md

```diff
@@ -26,11 +26,13 @@ If not already configured, install the [AWS integration][1]. This allows Datadog
 
 If you previously set up Datadog Serverless using the Datadog Forwarder, see the [installation instructions][2].
 
+If your Go Lambda functions are still using runtime [go1.x], consider either [migrating][3] to `provided.al2` or using the [Datadog Forwarder][2] instead of the Datadog Lambda Extension.
```

Comment:
```suggestion
If your Go Lambda functions are still using runtime `go1.x`, consider either [migrating][3] to `provided.al2` or using the [Datadog Forwarder][2] instead of the Datadog Lambda Extension.
```